shell/ci/testing/setup-devenv.sh

```diff
@@ -36,7 +36,7 @@ if [[ -n $CI ]]; then
     tmpFile=$(mktemp)
     keyringLocation=/etc/apt/keyrings/kubernetes-archive-keyring.gpg
 
-    curl -fsSLo "$tmpFile" https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    curl -fsSLo "$tmpFile" https://pkgs.k8s.io/core:/stable:/v1.29/deb/Release.key
```

Comment:
I think we should make `https://pkgs.k8s.io/core:/stable:/v1.29/deb/` a variable, so that we can use it below as well.